static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -144,15 +144,9 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
 
   renderTabs() {
     // TODO: Convert to styled component
-    const {organization} = this.props;
-    // TODO(cathy): remove feature check
-    const tabs =
-      this.provider.key in integrationFeatures &&
-      organization.features.filter(value =>
-        integrationFeatures[this.provider.key].includes(value)
-      )
-        ? this.tabs
-        : this.tabs.filter(tab => tab !== 'features');
+    const tabs = integrationFeatures.includes(this.provider.key)
+      ? this.tabs
+      : this.tabs.filter(tab => tab !== 'features');
```

Comment:
Do we need to include a test for this check?

Reply:
the test that i fixed covers this